app/src/main/java/com/vultisig/wallet/models/Chain.kt

```diff
@@ -125,11 +125,11 @@ val Chain.logo: Int
         Chain.solana -> R.drawable.solana
         Chain.ethereum -> R.drawable.ethereum
         Chain.avalanche -> R.drawable.avax
-        Chain.base -> R.drawable.eth_base
-        Chain.blast -> R.drawable.eth_blast
-        Chain.arbitrum -> R.drawable.eth_arbitrum
-        Chain.polygon -> R.drawable.eth_polygon
-        Chain.optimism -> R.drawable.eth_optimism
```

Comment:
Are old resources still in use after this PR? If not they should probably be deleted

Reply:
Yes
like https://github.com/vultisig/vultisig-android/blob/d42af0710fa2fc618ee75656cf260add3aacaf9e/app/src/main/java/com/vultisig/wallet/models/Coin.kt#L288